authorization/resourcetype/scope/repository/resource_type_scope.go

```diff
@@ -196,3 +199,44 @@ func (m *GormResourceTypeScopeRepository) List(ctx context.Context, resourceType
 	}
 	return rows, nil
 }
+
+/*
+
+// ListByName returns all resource type scopes filtered by name
+func (m *GormResourceTypeScopeRepository) ListByName(ctx context.Context, name string) ([]ResourceTypeScope, error) {
+	return m.Query(FilterByScopeName(name))
+}
+*/
+// ListByResourceTypeAndScope returns all resource type scopes filtered by name and scope.
+func (m *GormResourceTypeScopeRepository) ListByResourceTypeAndScope(ctx context.Context, resourceTypeID uuid.UUID, name string) ([]ResourceTypeScope, error) {
```

Comment:
What is the use case for this function?

Reply:
Can't seem to remember how this got here :) removing..
nice catch!

Reply:
they are actually used in the role model test https://github.com/fabric8-services/fabric8-auth/pull/347/files#diff-c033b4ea6b843569115b9d21a5249f39R169 , so I'll have to keep this function.

Reply:
It was added after your review comment https://github.com/fabric8-services/fabric8-auth/pull/347#discussion_r172490229

Reply:
Is it used outside of the tests though? I'm wondering what the purpose of this method is in production - to me, I think a Lookup() method that accepted the resource type and scope as parameters (so same as you have now), however returns the single scope record found, would be more useful.  How can there be multiple scopes returned if you're already providing both the resource type and the scope name?

Reply:
> I'm wondering what the purpose of this method is in production

It's not being exposed as a service, rather as a clean API - a wrapper over calling the raw sql.

> however returns the single scope record found, would be more useful

Good catch, let me fix this.

Reply:
https://github.com/fabric8-services/fabric8-auth/pull/347/commits/46fe4512e9a3a57b025f2f7a4e963d3d2205c0c5